AutomatedTesting/project.json

```diff
@@ -5,12 +5,15 @@
     "modules": [],
     "project_id": "{D816AFAE-4BB7-4FEF-88F4-E2B786DCF29D}",
     "android_settings": {
-        "package_name": "com.lumberyard.yourgame",
+        "package_name": "com.o3de.automatedtesting",
```

Comment:
Since its o3de.org, the package name here (and anywhere else that uses this convention) should be
```
org.o3de.automatedtesting
```

Reply:
yup. Thanks